### PR TITLE
nothing much just some formatting

### PR DESCRIPTION
**Pull request by codingwithcn**
took the time to deeply read through the code to know how things works

### CODE REVIEW

File: 2023-Season/src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -107,12 +107,23 @@ public void simulationPeriodic() {
...
+    var translationX = OneDimensionalLookup.interpLinear(
+      Constants.XY_Axis_inputBreakpoints,
+      Constants.XY_Axis_outputTable, 
+      _driverController.getLeftY()
+    ) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
+
+    var translationY = OneDimensionalLookup.interpLinear(
+      Constants.XY_Axis_inputBreakpoints,
+      Constants.XY_Axis_outputTable, 
+      _driverController.getLeftX()
```
Comment (Koltara) [inline 2023-Season/src/main/java/frc/robot/Robot.java]: I would argue this line is harder to read than before, you can move the math to be on the same line and keep each parameter separate.

File: 2023-Season/src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -107,12 +107,23 @@ public void simulationPeriodic() {
...
+    ) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
+
+    var rotationZ = OneDimensionalLookup.interpLinear(
+      Constants.RotAxis_inputBreakpoints, 
+      Constants.RotAxis_outputTable,
+      _driverController.getRightX()
+    ) * Constants.MAX_ANGULAR_VELOCITY_PER_SECOND;
```
Comment (Koltara) [inline 2023-Season/src/main/java/frc/robot/Robot.java]: Same as above.

File: 2023-Season/src/main/java/frc/robot/Utilities/Limelight.java [hunk 3]
```diff
@@ -5,60 +5,54 @@
...
-    _ledMode = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kLEDModeKey);
-    _camMode = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kCamModeKey);
-    _tagId = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kTargetTagId);
-}
-
+        _validTargets = NetworkTableInstance.getDefault()
+            .getTable(Constants.kTableName)
+            .getEntry(Constants.kValidTargetKey);
 
+        _ledMode = NetworkTableInstance.getDefault().getTable(Constants.kTableName).getEntry(Constants.kLEDModeKey);
```
Comment (Koltara) [inline 2023-Season/src/main/java/frc/robot/Utilities/Limelight.java]: We should be consistent when formatting similar code, so these should look like the above lines or vice versa.

File: 2023-Season/src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -107,12 +107,23 @@ public void simulationPeriodic() {
...
-        Constants.XY_Axis_outputTable, _driverController.getLeftY()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
-    var translationY = OneDimensionalLookup.interpLinear(Constants.XY_Axis_inputBreakpoints,
-        Constants.XY_Axis_outputTable, _driverController.getLeftX()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
-    var rotationZ = OneDimensionalLookup.interpLinear(Constants.RotAxis_inputBreakpoints, Constants.RotAxis_outputTable,
-        _driverController.getRightX()) * Constants.MAX_ANGULAR_VELOCITY_PER_SECOND;
+    var translationX = OneDimensionalLookup.interpLinear(
+      Constants.XY_Axis_inputBreakpoints,
+      Constants.XY_Axis_outputTable, 
+      _driverController.getLeftY()
+    ) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
```
Comment (Koltara) [inline 2023-Season/src/main/java/frc/robot/Robot.java]: I would argue this line is harder to read than before, you can move the math to be on the same line and keep each parameter separate.

**Review by Koltara — CHANGES_REQUESTED** [review]
Completed review with comments.

File: 2023-Season/src/main/java/frc/robot/Robot.java (hunk 2 above)
Comment (codingwithcn) [reply 2023-Season/src/main/java/frc/robot/Robot.java]: got you will move the math stuff on the same line

File: 2023-Season/src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (codingwithcn) [reply 2023-Season/src/main/java/frc/robot/Robot.java]: got you will move the math stuff on the same line

File: 2023-Season/src/main/java/frc/robot/Robot.java (hunk 4 above)
Comment (codingwithcn) [reply 2023-Season/src/main/java/frc/robot/Robot.java]: got you will move the math stuff on the same line

File: 2023-Season/src/main/java/frc/robot/Utilities/Limelight.java (hunk 3 above)
Comment (codingwithcn) [reply 2023-Season/src/main/java/frc/robot/Utilities/Limelight.java]: lol I totally forgot about changing this :)

**Comment by codingwithcn** [comment]
Made the changes requested

File: 2023-Season/src/main/java/frc/robot/Robot.java [hunk 5]
```diff
@@ -107,12 +107,20 @@ public void simulationPeriodic() {
...
-    var translationX = OneDimensionalLookup.interpLinear(Constants.XY_Axis_inputBreakpoints,
-        Constants.XY_Axis_outputTable, _driverController.getLeftY()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
-    var translationY = OneDimensionalLookup.interpLinear(Constants.XY_Axis_inputBreakpoints,
-        Constants.XY_Axis_outputTable, _driverController.getLeftX()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
-    var rotationZ = OneDimensionalLookup.interpLinear(Constants.RotAxis_inputBreakpoints, Constants.RotAxis_outputTable,
-        _driverController.getRightX()) * Constants.MAX_ANGULAR_VELOCITY_PER_SECOND;
+    var translationX = OneDimensionalLookup.interpLinear(
+      Constants.XY_Axis_inputBreakpoints,
+      Constants.XY_Axis_outputTable, 
+      _driverController.getLeftY() ) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
```
Comment (Koltara) [inline 2023-Season/src/main/java/frc/robot/Robot.java]:
```suggestion
      _driverController.getLeftY()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
```

File: 2023-Season/src/main/java/frc/robot/Robot.java [hunk 6]
```diff
@@ -107,12 +107,20 @@ public void simulationPeriodic() {
...
+
+    var translationY = OneDimensionalLookup.interpLinear(
+      Constants.XY_Axis_inputBreakpoints,
+      Constants.XY_Axis_outputTable, 
+      _driverController.getLeftX() ) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
```
Comment (Koltara) [inline 2023-Season/src/main/java/frc/robot/Robot.java]:
```suggestion
      _driverController.getLeftX()) * Constants.MAX_VELOCITY_METERS_PER_SECOND;
```

File: 2023-Season/src/main/java/frc/robot/Robot.java [hunk 7]
```diff
@@ -107,12 +107,20 @@ public void simulationPeriodic() {
...
+
+    var rotationZ = OneDimensionalLookup.interpLinear(
+      Constants.RotAxis_inputBreakpoints, 
+      Constants.RotAxis_outputTable,
+      _driverController.getRightX() ) * Constants.MAX_ANGULAR_VELOCITY_PER_SECOND;
```
Comment (Koltara) [inline 2023-Season/src/main/java/frc/robot/Robot.java]:
```suggestion
      _driverController.getRightX()) * Constants.MAX_ANGULAR_VELOCITY_PER_SECOND;
```

**Review by Koltara — CHANGES_REQUESTED** [review]
Almost done, just some spaces.

**Review by BigNoah37 — APPROVED** [review]
Thanks for cleaning up the code, Looks good after those changes were made.

**Review by Koltara — APPROVED** [review]
Looks good.

**Pull request merged**